scspell/__init__.py

```diff
@@ -556,13 +581,32 @@ def spell_check_file(filename, dicts, ignores, report_only, c_escapes):
             # This is matching the file-id.  Skip over it.
             pos = m_id.end()
             continue
-        result = spell_check_token(MatchDescriptor(data, m),
+        md = MatchDescriptor(data, m)
+        result = spell_check_token(md,
                                    filename, fq_filename, file_id_ref,
-                                   dicts, ignores, report_only)
+                                   dicts, ignores, report_only, list_unique)
+        prev = pos
         (data, pos) = result[0]
         error_found = result[1]
+
         if error_found:
             okay = False
+            if list_unique:
+                err = data[prev:pos]
+                if err in errors:
+                    errors[err].append(md.get_line_num())
+                else:
+                    errors[err] = [md.get_line_num()]
+
+    if list_unique:
+        errors = list(errors.items())
+        errors.sort(key=lambda x: x[1])
+
+        print('Set of mispelled tokens:')
```

Comment:
The rest of the code seems to print to `stderr`, so could you do the same here as well (`print(..., file=sys.stderr`)?